### ISSUE TITLE
CAPA machine pools autoscaling

### ISSUE THREAD

**Issue by alex-dabija** [giantswarm/roadmap#1798]
No description provided.

**Comment by AverageMarcus** [comment]
@alex-dabija It looks like all the tasks under this issue is now completed. Does this mean this issue is now resolved or is there still some more work that needs doing?

**Comment by alex-dabija** [comment]
> @alex-dabija It looks like all the tasks under this issue is now completed. Does this mean this issue is now resolved or is there still some more work that needs doing?

Yes, autoscaling support should be in CAPA. I think the cluster needs to be upgraded to version `v0.48.1`. @njuettner & @giantswarm/team-phoenix Could you confirm that this is the case?

**Comment by alex-dabija** [comment]
One bug was identified: https://github.com/giantswarm/giantswarm/issues/29030 .

**Comment by Rotfuks** [comment]
Need to adapt some last bits on the autoscaler app for the new EKS installations: giantswarm/giantswarm#29315
